src/app/app.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component } from '@angular/core';
+
+const context = require.context('../slides', true, /\.md|markdown$/);
+
+const slides: any[][] = context.keys()
```

Comment:
This is a shitty/slightly fragile "hack" that recursively loads all the markdown files in `src/slides` and makes them available to other components.

It works, though! The shittiest part is the `.sort` logic, which needs to be done to place something like `01.md` ahead of `01-a.md` and `01-b.md` in the array(s).